internal/pkg/cli/init.go

```diff
@@ -216,17 +216,17 @@ func (opts *InitAppOpts) createProject() error {
 }
 
 func (opts *InitAppOpts) createManifest() error {
-	manifest, err := manifest.Create(opts.AppName, opts.AppType)
+	manifest, err := app.Create(opts.AppName, opts.AppType)
```

Comment:
Thinking out loud here 🤔 :

At first glance, `app.Create` is not as obvious as `manifest.Create` in terms of what it's creating.  
What do you think about keeping the `manifest` pkg as is, but changing the function `Create` to `CreateApp`. This way you can have `manifest.CreateApp` or `manifest.CreatePipeline`.

Reply:
yea kinda like the idea